Peer/Program.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Peer.ConnectorApi;
+using Peer.Connectors;
+using Peer.Domain;
+using Peer.Domain.Models;
+using Peer.User;
+
+namespace Peer
+{
+    class Program
+    {
+        static async Task Main(string[] args)
+        {
+            IServiceCollection services = new ServiceCollection();
+
+            var config = new ConfigurationBuilder()
+                .AddJsonFile("appsettings.json", optional: true, reloadOnChange: true)
+                .AddEnvironmentVariables()
+                .Build();
+            var configModel = config.GetSection("UserConfig").Get<ConfigModel>();
+
+            // TODO: Create generic interface for IPullRequestSource
+            services.AddTransient<IPullRequestSource, GithubPullRequestSource>();
```

Comment:
You might want to handle conditionally registering types here. If you register AzDO and the AzDO config doesn't exist rn you'll get an explosion when you run which is less than ideal. Maybe you only want to register azure things if azure config exists?

Reply:
Made all the configs optional. Fixed the issue